app_dart/bin/validate_task_ownership.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cocoon_service/src/request_handlers/flaky_handler_utils.dart';
+import 'package:yaml/yaml.dart';
+
+/// Remote check based on flutter `repo` and the commit `ref`.
+///
+/// This currently supports `flutter/flutter` only.
+Future<List<String>> remoteCheck(String repo, String ref) async {
+  final HttpClient client = HttpClient();
+  final Uri ciYamlUrl = Uri.https('raw.githubusercontent.com', 'flutter/$repo/$ref/.ci.yaml');
+  final HttpClientRequest ciYamlRequest = await client.getUrl(ciYamlUrl);
+  final HttpClientResponse ciYamlResponse = await ciYamlRequest.close();
+  final Uri testOwnersUrl = Uri.https('raw.githubusercontent.com', 'flutter/$repo/$ref/TESTOWNERS');
```

Comment:
I have defined kTestOwnerPath in the flaky_handler_utils

Reply:
changed to refer there. Also the kCiYamlPath.